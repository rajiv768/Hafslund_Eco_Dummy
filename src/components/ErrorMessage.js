import React from 'react';

const ErrorMessage = () => {
  return (
    <div>
      <p>Sorry â€” something has gone wrong.</p>
    </div>
  );
};

export default ErrorMessage;
