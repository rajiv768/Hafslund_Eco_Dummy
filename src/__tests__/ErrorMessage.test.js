import React from 'react';
import '@testing-library/react/cleanup-after-each';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import ErrorMessage from '../components/ErrorMessage';

afterEach(cleanup);

describe('<ErrorMessage />', () => {
  it('renders as expected', async () => {
    const { container } = render(<ErrorMessage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
