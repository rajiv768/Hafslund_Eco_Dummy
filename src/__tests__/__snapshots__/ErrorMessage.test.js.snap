// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> renders as expected 1`] = `
<div>
  <p>
    Sorry â€” something has gone wrong.
  </p>
</div>
`;
